testdata/sqllogictests/functions/kdl.slt

```diff
@@ -2,8 +2,9 @@ statement ok
 create table cows (id int, docs string);
 
 statement ok
-
 insert into cows values (0, 'name "betsy" status="amazing" age=89; details "classic" a=1 b=100;');
+
+statement ok
 insert into cows values (1, 'name "zabby" status="electric" age=120; details "zesty" a=1 b=400;');
```

Comment:
sql logic tests explicitly forbid multiple statements per block. FlightSQL couldn't properly process these transactions, and since it's not technically allowed in slt's, i just changed this to multiple statements.